controllers/v1/observationsController.js

```diff
@@ -1796,4 +1797,109 @@ module.exports = class Observations extends Abstract {
         });
     }
 
+    /**
+    * @api {get} /assessment/api/v1/observations/getObservationSolutionLink?observationsolutionId:observationsolutionId&appName:appName 
+    * @apiVersion 1.0.0
```

Comment:
@priyanka-TL  This api should be in solution. As we are adding link in solution not in observation.